pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -469,9 +469,14 @@ func (b *volumeBinder) checkBindings(pod *v1.Pod, bindings []*bindingInfo, claim
 		return false, fmt.Errorf("failed to get cached claims to provision for pod %q", podName)
 	}
 
-	node, err := b.nodeInformer.Lister().Get(pod.Spec.NodeName)
-	if err != nil {
-		return false, fmt.Errorf("failed to get node %q: %v", pod.Spec.NodeName, err)
+	var node *v1.Node
```

Comment:
I recall we will have some means to associate the node and the RP, so we won't need to end up looping through them to find the node by names.

Reply:
The association is in NodeInfo and podState. Neither of them were passed into this function. Keep the looping now to pass the compilation w/o further changes.

> I recall we will have some means to associate the node and the RP, so we won't need to end up looping through them to find the node by names.